saleor/graphql/shop/types.py

```diff
@@ -293,6 +294,10 @@ class Shop(graphene.ObjectType):
             AuthorizationFilters.AUTHENTICATED_APP,
         ],
     )
+    schema_version = graphene.String(
+        description="Minor Saleor API version." + ADDED_IN_39,
```

Comment:
Hm, I'm not sure if `ADDED_IN_39` should be here, as it will be porting to 3.5 version.

Reply:
Should I change it to 3.5 or get rid of it?

Reply:
@patrys can you advise?

Reply:
It should state the first minor release where this is/was added. In this case, that would be 3.5.